operations/reservations.md

```diff
@@ -820,6 +820,15 @@ Marks a reservation as `Started` \(= checked in\). Succeeds only if all starting
 
 Marks a reservation as `Processed` \(= checked out\). Succeeds only if all processing conditions are met \(the reservation has the `Started` state, balance of all reservation members is zero etc\).
 
+### Conditions
+
+- Reservation has already been checked in.
+- Reservation hasn't been checked out previously.
+- Reservation can't be checked out sooner than last day of planned stay.
+- The companion profiles of reservation are complete.
```

Comment:
Yeah, they receive message like this: `"Message": "Can't perform operation, because 1 out of 2 companions don't have a customer profile.\nReservation can't be checked in or out because the following customer profiles are missing required information: Customer name.",`